webrtc/go/go.orig.mod

```diff
@@ -0,0 +1,126 @@
+module blah
+
+replace github.com/libp2p/go-libp2p => /go-libp2p
+
+require (
+	github.com/libp2p/go-libp2p v1.0.5
+	github.com/libp2p/go-libp2p-core v0.20.1
+	github.com/testground/sdk-go v0.3.0
+	golang.org/x/sync v0.0.0-20220819030929-7fc1605a5dde
+)
+
+require (
+	github.com/avast/retry-go v2.6.0+incompatible // indirect
```

Comment:
That are really a lot of direct requirements o.O Looks more like a `go.sum` file (length wise) than a go.mod file. You do not rely on these directly, right? So why mention them all here instead of only the ones you're importing?

Reply:
There are 4 direct requirements.  I'm 100% sure 2 of them are necessary. I guess I could experiment with removing the other 2 if you'd like.

The `// indirect` stuff was generated by... I think `go mod tidy` maybe?

Reply:
well the only ones required are the ones you import yourself. The indirect stuff is indeed possible, bit annoying though, but I guess it is what it is :P